src/ploigos_step_runner/step_implementers/create_container_image/buildah.py

```diff
@@ -100,6 +102,35 @@ def _required_config_or_result_keys():
         """
         return REQUIRED_CONFIG_OR_PREVIOUS_STEP_RESULT_ARTIFACT_KEYS
 
+    def get_file_hash(self, application_name, service_name, tar_file):
```

Comment:
similar funciton in rekor. can we have one fuction that lifes in utils/files.py that both rekor and buildah can use?

Reply:
nvm. change this to be `get_image_sha` and use pythong to extract the *.json file that isn't the `manifsest.json` file and do a sha256sum on it.

Reply:
repalce all this with `buildah --storage-driver=vfs images --digests --format "{{.Digest}}" IMAGE_REF`